posts/frangeris/01.reusable-dynamic-modal-on-vue3.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Reusable dynamic modal on Vue 3
+published: false
+description: Quick explanation on how to make use of new composition API on Vue 3 to get a reusable modal
+tags: "javascript, vue, webdev, tutorial"
+cover_image: ./assets/01.cover.png
+---
+
+## Reusable dynamic modal on Vue 3
+
+Most of the time on frontend development the best way to keep a consistent way of building components is to try to make them reusable every time we can, but sometimes the framework itself can make it a bit harder if we don’t have deep knowledge of its internal API, specifically the way it handle view instance and state component data.
```

Comment:
```suggestion
When working on frontend development, most of times the best way to keep a consistent way of building components is to try to make them reusable every time we can, but sometimes the framework itself can make it a bit harder if we don’t have deep knowledge of its internal API, specifically the way it handle view instance and state component data.
```

Reply:
Prefiero el `most of the time` por la forma en como quiero que se interprete, agregando el `when working` lo hace redundante, gracias igual 🙌🏾